draft-thomson-webpush-http2.xml

```diff
@@ -109,6 +107,56 @@
         overhead costs.
       </t>
       <t>
+        A push server that does not support reliable delivery over intermittent
+        network connections or failing applications on devices, forces the
+        device to acknowledge receipt directly to the application server,
+        incurring additional power drain in order to establish (usually secure)
+        connections to the individual application servers.
+      </t>
+      <t>
+        While reliability is not required for messages that expire in few
+        seconds (e.g. an incoming call) or collapsible ones (e.g. the current
+        number of unread emails), it is more important when messages contain
```

Comment:
Note: we haven't introduced collapsible messages at this point so mentioning them might be confusing.